themes/theme-gmd/components/AppBar/presets/BackBar/index.jsx

```diff
@@ -30,6 +38,7 @@ function BackBar({ goBack, ...props }) {
 
 BackBar.propTypes = {
   goBack: PropTypes.func.isRequired,
+  prevTitle: PropTypes.string.isRequired,
```

Comment:
Since the prop is required, validation errors come visible when the previous route doesn't have a title (e.g. "page" or deeplink)